### PR TITLE
Visual indicator for overdue reminders and test cases for meetings and reminders commands

### LINKED ISSUE #110: As a visual person, I can quickly identify overdue reminders
so that I can work on it without further delay

### PR DESCRIPTION
**Pull request by sebastiantoh**
Closes #110, #60 

- Enabled assertions in gradle (required for the coming week's tutorial)
- Added a visual indicator for overdue reminders
  - Reminders which are overdue will have its date label in red instead of white
- Updated the command feedback from running `reminder list` so that it will not display the entire reminder list in the chatbox.
- Extracted ModelStub from a private class in `seedu.address.commands.contact.AddCommandTest` to a public class in `seedu.address.model`
- Added tests cases for `commands.reminder` and `commands.meeting`

Demo:
image

### CODE REVIEW

**Review by hakujitsu — COMMENTED** [review]
Tested locally, LGTM! Good idea to abstract ModelStub out to another class.

Can I just check: whether a reminder is overdue or not is evaluated at program startup, are there any plans to either have the reminder display update immediately when the reminder time has passed, or to have a command to manually refresh the reminder display (like perhaps on `reminder list`?

**Comment by sebastiantoh** [comment]
> 
> 
> Tested locally, LGTM! Good idea to abstract ModelStub out to another class.
> 
> Can I just check: whether a reminder is overdue or not is evaluated at program startup, are there any plans to either have the reminder display update immediately when the reminder time has passed, or to have a command to manually refresh the reminder display (like perhaps on `reminder list`?

Good point! I've added a force-refresh at the end of every command. Idk if this would be too inefficient though...

**Comment by hakujitsu** [comment]
I think that should be fine, we could relook it if the refreshes cause any issues later!

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 1]
```diff
@@ -207,6 +207,12 @@ private CommandResult executeCommand(String commandText) throws CommandException
                 chatBox.clear();
             }
 
+            // Force refresh of the following UI components which are time sensitive
+            // Overdue reminders should be displayed differently
+            this.reminderListPanel.refresh();
+            // Past meetings should be filtered out
```
Comment (hakujitsu) [inline src/main/java/seedu/address/ui/MainWindow.java:213]: Is this something that was implemented in this PR? I didn't experience any filtering out of past meetings when executing commands to refresh the display.

**Review by hakujitsu — COMMENTED** [review]
Just checking: is the meeting panel refresh is implemented in this PR too?

File: src/main/java/seedu/address/ui/MainWindow.java (hunk 1 above)
Comment (sebastiantoh) [reply src/main/java/seedu/address/ui/MainWindow.java:213]: Not yet! Just adding this in here so that I won't forget. Sorry for the confusion

Comment (hakujitsu) [reply src/main/java/seedu/address/ui/MainWindow.java:213]: Great, thanks for clarifying! Will approve and merge.

**Review by hakujitsu — APPROVED** [review]

**Pull request merged**